docs/redirects/training.md

```diff
@@ -1,9 +1,8 @@
 ---
-title: "项目构建设置"
-nav-id: project-setup
-nav-title: '项目构建设置'
-nav-parent_id: getting-started
-nav-pos: 30
+title: Hands-on Training
+layout: redirect
+redirect: /learn-flink/html
```

Comment:
This redirect is very confused. learn-flink/html isn't a page one to redirect to, and it has no relationship to index.zh.md.